testproject/Assets/Tests/Manual/Scripts/ManualNetworkVariableTest.cs

```diff
@@ -1,5 +1,6 @@
 using System;
```

Comment:
I have no idea how to run this

Reply:
This was added when we had about 0 testing abilities. :-)
The way to test this is to add a GameObject to your scene, add a ManualNetworkVariableTest component to it, and also add a NetworkObject component to your game object. Then, run a client-host game (in actual different instances, outside Editor, like the real shipping game), and check the logs.

As it turns out, it is broken (in this branch), but there's a good chance it's not your changes, and that it just broke in `develop`. I'll chase this and let you know either way.

Error I'm currently getting are:
```
InvalidOperationException: Cannot access NetworkList before it's initialized
  at MLAPI.NetworkVariable.Collections.NetworkList`1[T].EnsureInitialized () [0x0000e] in <fd922b06195a430a872f1c18ad7a2f2a>:0
  at MLAPI.NetworkVariable.Collections.NetworkList`1[T].Add (T item) [0x00000] in <fd922b06195a430a872f1c18ad7a2f2a>:0
  at TestProject.ManualTests.ManualNetworkVariableTest.FixedUpdate () [0x0002c] in <c508130d0af74c0eba930e4ec0acedc6>:0
  ```
(will investigate)

Reply:
The test has a bug I could fix, but then:

```               
// MSW: This test relies on the LocalTick variable which was otherwise dormant
//  and remove from MLAPI.
```
So don't worry about it